devel/cutter/Portfile

```
@@ -15,13 +15,16 @@ long_description    Cutter is a xUnit family Unit Testing Framework for C and C+
                     It provides easy to write test API, useful output format for debugging \
                     and so on.
 
-homepage            http://cutter.sourceforge.net/
+homepage            https://cutter.sourceforge.net/
 platforms           darwin
 license             LGPL-3+
 
-master_sites        sourceforge:project/cutter/cutter/${version}
-checksums           rmd160  e744ea0374222c32b7ba5d92ae4279908ba35216 \
-                    sha256  b917a5b67b1d5f24458db5ab177dc11d23b47f04f9cb7eef757f456483c629c6
+master_sites        https://osdn.dl.osdn.net/cutter/73761/ \
```

MacPorts is already aware of `master_sites` from SourceForge.

I think you should be able to do something like:

```suggestion
master_sites        sourceforge:cutter/73761
```

...instead of manually specifying a selection of mirrors.

---

Let me try.

---

> MacPorts is already aware of `master_sites` from SourceForge.
> 
> 
> 
> I think you should be able to do something like:
> 
> 
> 
> ```suggestion
> 
> master_sites        sourceforge:cutter/73761
> 
> ```
> 
> ...instead of manually specifying a selection of mirrors.

@herbygillot 
Is it even on SourceForge now? Project page is https://osdn.net/projects/cutter
Seems that from SF it is just a redirect.